docs/workshop/intro_to_project.md

```diff
@@ -0,0 +1,23 @@
+# Introduction
+
+## Introduction to AIOps
+
+![Intro to AIOps](../assets/images/intro_to_aiops.png)
+
+AIOps stands for Artificial Intelligence for IT Operations. AIOps is a critical component of supporting any Open Hybrid Cloud infrastructure. In our opinion,  it’s mostly a cultural change, very much as we saw with the devops movement.Here we combined the two cultural mentalities from dev and ops to create a new culture, the devops culture. And the same cultural change is going to happen with AIOps. The devops culture and the data science culture are using different tools and have some different mindsets.
```

Comment:
I think this is good to speak out with support of slides. If we are creating documentation, I would make it more generic and less opinionated.

```suggestion
AIOps stands for Artificial Intelligence for IT Operations. AIOps is a critical component of supporting any Open Hybrid Cloud infrastructure. AIOps involves using big data processing, machine learning and AI tooling to support IT operations. Some common examples of AIOps tools are anomaly detection, correlation analysis, time-to-response prediction, log analysis. Tools such as these can help facilitate collaboration between data science practitioners and DevOps engineers and help with streamlining incident management, issue resolution and faster root cause analysis.
```